### PR TITLE
Allows easy hosting of server side lobby music

### PR DESCRIPTION
**Pull request by j-mie**
Same as #24563 but for music

🆑 Jambread/RemieRichards/Incoming5643
server: There is a new system for title music accessible from config/title_music folder. This system allows for multiple rotating lobby music without bloating Git as well as map specific lobby music. See the readme.txt in config/title_music for full details.
config: The previous method of title music selection, strings/round_start_sounds.txt has been depreciated by this change.
/🆑

### CODE REVIEW

File: config/title_music/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+---EXAMPLES (NOT PART OF ANY LICENSE)---
+
+These are examples of properly attrubuted and licensed sounds. 
+They are not an actual part of any license under any circumstance.
+
+title5.ogg was created by Mya Quinn on Feburary 28, 2557. It is licensed under a Combative Clowning 3.0 HO-NK license (http://example.com/license/url/). 
+
+Unless otherwise noted all sounds were created by Cuban Pete on July 26, 2555. They are licensed under the RUMBABEAT Public License.(http://example.com/license/url/).
```
Comment (praisenarsie) [inline config/title_music/LICENSE.txt]: chiki chiki boom boom

File: code/game/area/Space_Station_13_areas.dm [hunk 2]
```diff
@@ -41,7 +41,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	power_environ = FALSE
 	valid_territory = FALSE
 	outdoors = TRUE
-	ambientsounds = list('sound/ambience/ambispace.ogg','sound/ambience/title2.ogg')
```
Comment (Cyberboss) [inline code/game/area/Space_Station_13_areas.dm]: Keep dis

File: code/game/area/areas/ruins/space.dm [hunk 3]
```diff
@@ -22,7 +22,7 @@
 	power_environ = FALSE
 	valid_territory = FALSE
 	outdoors = TRUE
-	ambientsounds = list('sound/ambience/ambispace.ogg','sound/ambience/title2.ogg')
```
Comment (Cyberboss) [inline code/game/area/areas/ruins/space.dm]: And dis

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Existing title songs should be moved to the new config folder as defaults. They can't be removed from git history anyway

**Review by Cyberboss — DISMISSED** [review]

File: code/controllers/subsystem/ticker.dm [hunk 4]
```diff
@@ -66,10 +66,56 @@ SUBSYSTEM_DEF(ticker)
...
+
 	var/old_login_music = trim(file2text("data/last_round_lobby_music.txt"))
 	if(music.len > 1)
 		music -= old_login_music
+
+	for(var/S in music)
+		var/list/L = splittext(S,".")
+		if(L.len >= 2)
+			var/ext = lowertext(L[L.len]) //pick the real extension, no 'honk.ogg.exe' nonsense here
+			var/static/list/byond_sound_formats = list(
```
Comment (Cyberboss) [inline code/controllers/subsystem/ticker.dm]: don't make this a static list since it's only used once

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/ticker.dm [hunk 5]
```diff
@@ -66,10 +66,56 @@ SUBSYSTEM_DEF(ticker)
...
+
 	var/old_login_music = trim(file2text("data/last_round_lobby_music.txt"))
 	if(music.len > 1)
 		music -= old_login_music
+
+	for(var/S in music)
+		var/list/L = splittext(S,".")
+		if(L.len >= 2)
+			var/ext = lowertext(L[L.len]) //pick the real extension, no 'honk.ogg.exe' nonsense here
+			var/list/byond_sound_formats = list(
```
Comment (Cyberboss) [inline code/controllers/subsystem/ticker.dm]: Create this list outside the for

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**